spark-board-ui/src/column.jsx

```diff
@@ -7,14 +7,14 @@ import { Handle, Position } from 'reactflow';
 import "./column.css";
 
 function ColumnNode({ data }) {
-    const nodeStyle = {};
-
     return (
-        <div className="column-node__container" style={ nodeStyle } >
-            <Handle type="target" position={Position.Left} id="target" />
-            <p>{ data.name }</p>
-            <Handle type="source" position={Position.Right} id="source" />
-        </div>
+        <>
+            <div className="column-node__container" >
+                <Handle type="target" position={Position.Left} id="target" />
+                <p>{ data.name }</p>
+                <Handle type="source" position={Position.Right} id="source" />
+            </div>
+        </>
```

Comment:
I don't see why this change is necessary

Reply:
Reverted